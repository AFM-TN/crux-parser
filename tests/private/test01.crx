// test empty function definition

func main() : void {
}
