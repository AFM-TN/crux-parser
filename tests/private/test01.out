PROGRAM
  DECLARATION_LIST
    DECLARATION
      FUNCTION_DEFINITION
        PARAMETER_LIST
        TYPE
        STATEMENT_BLOCK
          STATEMENT_LIST